Reportedly/Services/Network/RESTful/API/AuthenticationAPIService.swift

```diff
@@ -62,23 +70,21 @@ final class AuthenticationAPIService: BaseAPIService, AuthenticationAPIServicePr
             bodyParams: signupRequest.toDictionary(),
             shouldAuthenticate: false
         ) { data, success, error in
-            Thread.executeOnMainThread {
-                if let error = error {
-                    return completion(.failure(error))
-                }
-                guard let success = success else {
-                    return completion(.failure(ResponseError(.other)))
-                }
-                switch success.type {
-                case .noContent: completion(.success(ResponseSuccess(.noContent)))
-                case .success: completion(.success(ResponseSuccess(.success)))
-                }
+            if let error = error {
```

Comment:
Can we apply `typealias Result = Alamofire.Result`? , then we can return the response directly. What do you think? 😅

Reply:
yeah, we will refactor the network layer in the next Chore pull request 🙏